Assets/Scripts/CreateQRCode.cs

```diff
@@ -0,0 +1,49 @@
+﻿using UnityEngine;  
+using System.Collections;  
+using ZXing; 
+using ZXing.QrCode;  
+using UnityEngine.UI;  
+
+public class CreateQRCode : MonoBehaviour  
```

Comment:
I think we do not need this file.
because , we don't have to create QR Coed on our app.